src/frameworks/us_privacy_string_and_api_generator.js

```diff
@@ -0,0 +1,147 @@
+const FrameworkBase = require('./base');
+
+const US_PRIVACY_API_VERSION = 1;
+
+/**
+ * Implements usprivacy string framework
+ * for more information on the US Privacy string see:
+ * https://github.com/InteractiveAdvertisingBureau/USPrivacy/blob/master/CCPA/US%20Privacy%20String.md#us-privacy-string-format
+ */
+class UsPrivacyStringAndAPIGenerator extends FrameworkBase {
+  constructor() {
+    super();
+    this.window = window;
+    this.document = document;
+  }
+  supportedGenerators() {
```

Comment:
(MINOR) Probably want a newline between L15 & L16

Reply:
doh, thank you!